docs/content/policy-reference.md

```diff
@@ -824,6 +824,48 @@ Timezones can be specified as
 
 Note that the opa executable will need access to the timezone files in the environment it is running in (see the [Go time.LoadLocation()](https://golang.org/pkg/time/#LoadLocation) documentation for more information).
 
+Below is an example that uses some of the Time built-in functions.
+
+This package will check if a container image is older than 2 months. 
```

Comment:
It also checks if the image is older than 1 year, right? I'm not sure I understand why you'd want both conditions.